src/browser/BrowserService.cpp

```diff
@@ -317,7 +320,7 @@ QString BrowserService::storeKey(const QString& key)
             return {};
         }
 
-        contains = db->metadata()->customData()->contains(QLatin1String(ASSOCIATE_KEY_PREFIX) + id);
+        contains = db->metadata()->customData()->contains(QLatin1String(ASSOCIATE_KEY_PREFIX.toUtf8()) + id);
```

Comment:
I think `QLatin1String(ASSOCIATE_KEY_PREFIX.toUtf8())` can simply be `ASSOCIATE_KEY_PREFIX`?